src/planscape/planning/views.py

```diff
@@ -252,7 +254,10 @@ def list_planning_areas(request: HttpRequest) -> HttpResponse:
     Retrieves all planning areas for a user.
     Requires a logged in user.  Users can see only their owned planning_areas.
 
-    Returns: A list of planning areas in JSON form.
+    Returns: A list of planning areas in JSON form.  Each planning area JSON will also include
+        two metadata fields:
+      scenario_count: number of scenarios for the planning area returned.
+      scenario_latest_updated_at: latest datetime across all scenarios or None if there are no scenarios.
```

Comment:
Can we add what specific format the time stamp will be stored in?

Reply:
Done.  (It's, well, datetime.)

Reply:
Oh lol, sorry my brain just read "datetime" as "timestamp" for some reason. Thanks for adding the example!